docs-client/webpack.config.ts

```diff
@@ -98,6 +99,12 @@ const config: Configuration = {
     new DefinePlugin({
       'process.env.WEBPACK_DEV': JSON.stringify(process.env.WEBPACK_DEV),
     }),
+    new CompressionWebpackPlugin({
+      test: /\.js(\?.*)?$/i,
+      deleteOriginalAssets: !isDev,
+      algorithm: 'gzip',
+      filename: '[path].gz',
```

Comment:
Should we use `br` assuming most modern browsers already supports Brotli? Which yields smaller files?

Reply:
Here's what it looks like:

<img width="780" alt="스크린샷 2022-01-14 오후 10 13 22" src="https://user-images.githubusercontent.com/22594101/149520822-dea0ebfd-c99d-4dec-bcdd-b45f4f7bd2a6.png">

```typescript
new CompressionWebpackPlugin({
      test: /\.js(\?.*)?$/i,
      deleteOriginalAssets: !isDev,
      algorithm: 'gzip',
      filename: '[path].gz',
})
```

```typescript
new CompressionWebpackPlugin({
      test: /\.js(\?.*)?$/i,
      deleteOriginalAssets: !isDev,
      algorithm: 'brotliCompress',
      filename: '[path].br',
})
```

Reply:
`br` seems to work for 94+% of population already according to this: https://caniuse.com/brotli Let's switch to `br`.